dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -2096,6 +2105,201 @@
         "text"
       ]
     },
+    "AnalyzeConversationalSentimentTask": {
+      "type": "object",
+      "description": "Task definition for a sentiment analysis in conversations.",
+      "properties": {
+        "parameters": {
```

Comment:
Is this required? If so, be sure to call that our in the `required` properties.

Reply:
Same for any required properties below.

Reply:
Good catch! It is not required as it's a kind of analysis task. Will update it